apps/sharemyhealth/resources.py

```diff
@@ -0,0 +1,9 @@
+from social_django.models import UserSocialAuth
+
+
+class Resource(object):
+    """A python wrapper around the social_django.models.UserSocialAuth class."""
+    model_class = UserSocialAuth
+
```

Comment:
Follow the [pattern from django social auth for a name](https://github.com/TransparentHealth/smh_app/blob/master/apps/sharemyhealth/backends.py#L13)